packages/victory-core/src/victory-util/data.js

```diff
@@ -91,6 +92,72 @@ function cleanData(dataset, props) {
   });
 }
 
+// This method will remove data points that fall outside of the desired domain (non-continuous charts only)
+function formatDataFromDomain(dataset, props) {
+  const { domain, symbol, interpolation } = props;
+
+  if (!domain || interpolation) return dataset;
```

Comment:
I like using the existence of the `interpolation` prop as the check for continuous charts. Please add a comment, though.

Reply:
done